otter/integration/tests/test_launch_stack.py

```diff
@@ -0,0 +1,143 @@
+"""
+Tests covering Otter's integration with Heat for the launch_stack launch
+configuration.
+"""
+
+from pyrsistent import pbag
+
+from twisted.internet import reactor
+from twisted.internet.defer import inlineCallbacks
+from twisted.python.log import msg
+from twisted.trial import unittest
+from twisted.web.client import HTTPConnectionPool
+
+from otter.convergence.model import get_stack_tag_for_group
+
+from otter.integration.lib.autoscale import ScalingGroup, ScalingPolicy
+from otter.integration.lib.resources import TestResources
+from otter.integration.lib.trial_tools import (
+    TestHelper,
+    get_identity,
+    get_resource_mapping,
+    region)
+
+from otter.log import log
+
+from otter.util.deferredutils import retry_and_timeout
+from otter.util.http import check_success, headers
+from otter.util.logging_treq import LoggingTreq
+from otter.util.retry import (
+    TransientRetryError,
+    repeating_interval,
+    terminal_errors_except
+)
+
+
+class TestLaunchStack(unittest.TestCase):
+    """Tests making sure launch_stack launch configurations can be used."""
+
+    def setUp(self):
+        """
+        Establish an HTTP connection pool and commonly used resources for each
+        test. The HTTP connection pool is important for maintaining a clean
+        Twisted reactor.
+        """
+        self.helper = TestHelper(self)
+        self.rcs = TestResources()
+        self.identity = get_identity(self.helper.pool)
+        self.pool = HTTPConnectionPool(reactor, False)
+        self.treq = LoggingTreq(log=log, log_response=True)
+
+        scaling_group_config = {
+            'launchConfiguration': {
+                'args': {
+                    'stack': {
+                        'template': {
+                            'heat_template_version': '2015-04-30',
+                            'resources': {
+                                'rand': {'type': 'OS::Heat::RandomString'}
+                            }
+                        }
+                    }
+                },
+                'type': 'launch_stack'
+            },
+            'groupConfiguration': {
+                'name': 'test_launch_stack',
+                'cooldown': 0,
+                'minEntities': 0,
+                'maxEntities': 10
+            },
+            'scalingPolicies': [],
+        }
+
+        self.group = ScalingGroup(group_config=scaling_group_config,
+                                  treq=self.treq,
+                                  pool=self.pool)
+
```

Comment:
Non-blocking: Would it be possible to use [`TestHelper.create_group`](https://github.com/rackerlabs/otter/blob/master/otter/integration/lib/trial_tools.py#L208) instead? It might be complicated to modify that for `launch_stack` config? How about `create_stack_group` function that takes `stack_args` as argument?